product_docs/docs/edbcloud/beta/reference/index.mdx

```diff
@@ -205,10 +210,63 @@ The response of this API call includes the `access_token`. For example:
 
 
 
+## Using the get-token Script
```

Comment:
Also, this should be called something like "Request Your Token Using `get-token.sh`, right? To mirror the other heads? Same on L236 below.